.changeset/cold-apes-play.md

```diff
@@ -0,0 +1,7 @@
+---
+"@latticexyz/cli": patch
+"@latticexyz/world-modules": patch
+"@latticexyz/world": patch
+---
+
+Added a new module, `DelegationWithSignatureModule`, which allows registering delegations with a signature.
```

Comment:
```suggestion
Added a new preview module, `Unstable_DelegationWithSignatureModule`, which allows registering delegations with a signature.

Note: this module is marked as `Unstable`, because it will be removed and included in the default `World` deployment once it is audited.
```